src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -1556,7 +1558,35 @@ public boolean hasAlly(Town town) {
 	public boolean hasMutualAlly(Town town) {
 		return hasAlly(town) && town.hasAlly(this);
 	}
+	/**
+	 * Only to be used when loading the database.
+	 * @param towns List&lt;Town&gt; which will be loaded in as trusted towns.
+	 */
+	public void loadTrustedTowns(List<Town> towns) {
+		for (Town trustTown : towns) {
+			trustedTowns.put(trustTown.getUUID(), trustTown);
+		}
+	}
```

Comment:
```suggestion
	public void loadTrustedTowns(Iterable<? extends Town> towns) {
		towns.forEach(trustTown -> trustedTowns.put(trustTown.getUUID(), trustTown));
	}
```

First we just use a the functionality of an Iterable, so we can allow to just parse in an Iterbale.
Secondly we can generalize the generic, so we also accept a Iterable which includes objects that are just extending the Town.
Lastly we can use a forEach loop instead of a for loop.